modules/account-lib/src/coin/baseCoin/ed25519KeyPair.ts

```diff
@@ -74,4 +74,34 @@ export abstract class Ed25519KeyPair implements BaseKeyPair {
 
   /** @inheritdoc */
   abstract getKeys(): any;
+
+  /**
+   * Generates a signature for an arbitrary string with the current private key using the ed25519 public-key signature
+   * system.
+   *
+   * @param {string} message to produce a signature for
+   * @returns {Uint8Array} The signature produced for the message
+   * @throws if there is no private key
+   */
+  signMessage(message: string): Uint8Array {
```

Comment:
IMO the interface should either both take in uint8arrays as params and return uint8arrays as results, or take hex strings and return hex strings - its all just bytes so seems they should be represented the same way go avoid confusion IMO